ironfish/src/primitives/noteEncrypted.ts

```diff
@@ -22,9 +22,16 @@ export class NoteEncrypted {
 
   private noteEncrypted: NativeNoteEncrypted | null = null
   private referenceCount = 0
-
-  constructor(noteEncryptedSerialized: Buffer) {
+  /**
+   * Whether the note has already been previously validated, and thus does not
+   * need to be checked anymore after parsing. Used to speed up construction of
+   * `NativeNoteEncrypted` in `takeReference`.
+   */
+  private validated: boolean
```

Comment:
nit: I sorta wish this name matched the underlying rust name which is `skipValidation` but I'm not strongly opinionated about this.

Reply:
Changed every name to be in line with `skipValidation`